docs/src/main/sphinx/object-storage/legacy-s3.md

```diff
@@ -128,6 +128,12 @@ setting AWS access and secret keys in the `hive.s3.aws-access-key`
 and `hive.s3.aws-secret-key` settings, and also allows EC2 to automatically
 rotate credentials on a regular basis without any additional work on your part.
 
+If you are running Trino on Amazon EKS, and authenticate using a Kubernetes
+service account, you can set the
+`trino.s3.use-web-identity-token-credentials-provider` to `true`, so Trino does
```

Comment:
there's no config like this in Trino legacy S3 FS.

So the docs (and release notes) are wrong. @nineinchnick can you please update / fix this (both docs and adding a config to expose setting the hadoop config to users).

Reply:
https://github.com/trinodb/trino/pull/23544